package.json

```diff
@@ -41,7 +42,11 @@
     "webpack-merge": "^4.1.1"
   },
   "dependencies": {
+    "express": "^4.16.2",
+    "file-loader": "^1.1.6",
     "phaser": "3.0.0-beta.14",
-    "webfontloader": "^1.6.28"
+    "webfontloader": "^1.6.28",
+    "webpack-dev-middleware": "^2.0.3",
+    "webpack-hot-middleware": "^2.21.0"
```

Comment:
The `file-loader` is a production dependency. Also we should have a discussion about the dependencies (in an issue or chat preferably), because the current state is broken if you use the `--production` at install (e.g. webpack is required at build time).